app/views/works/index.html.erb

```diff
@@ -7,39 +7,37 @@
 <!-- /end descriptions-->
 
 <!--Subnavigation, sorting and actions.-->
-<% if @collection || @tag || @user %>
+<% if @collection || @tag || @user || @language %>
   <div class="navigation actions module">
     <h3 class="landmark heading"><%= ts("Navigation and Actions") %></h3>
     <!--gift search subnav-->
     <% if @collection && @collection.gift_exchange? %>
       <%= render 'gifts/gift_search' %>
     <% end %>
     <!--user role subnav-->
-    <% if @tag || @user || @collection %>
-      <ul class="user navigation actions" role="navigation">
-        <% if @user %>
-          <% if @user == current_user %>
-            <li id="edit_multiple"><%= link_to ts('Edit Works'), show_multiple_user_works_path(@user) %></li>
-          <% end %>
-          <li><%= span_if_current ts('Works in Collections'), collected_user_works_path(@user) %></li>
-          <li><%= span_if_current ts('Works'), user_works_path(@user) %></li>
+    <ul class="user navigation actions" role="navigation">
```

Comment:
Diff is more impressive than it is: I just removed the duplicate if (same condition checked as its parent above).